CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Welcome
+
+You have stumbled upon a top secret part of the internet where rocket scientists hang out.
+
+It would be a multi-national security concern if you joined a [Discord channel](https://discord.gg/sWdSk6m) where the rocket scientists answer any questions thrown their way.
+
+Of course we are kidding, Space Cloud welcomes you!
+
+## How to Contribute to Space Cloud? (General Guidelines)
```

Comment:
We could also include that contributing to the discussions and offering their views and feedback on the community calls is a welcome contribution

Reply:
This makes a lot of sense. We should definitely add this